tracer/test/snapshots/OwinWebApi2Tests.WithExpansion.__path=_api2_transientfailure_false_statusCode=500.verified.txt

```diff
@@ -41,18 +41,23 @@ at Samples.AspNetMvc5.Controllers.ConventionsController.TransientFailure(String
   {
     TraceId: Id_3,
     SpanId: Id_4,
-    Name: CustomTracingExceptionHandler.handle-async,
+    Name: CustomTracingExceptionHandler.internal,
     Resource: CustomTracingExceptionHandler.handle-async,
     Service: Samples.Owin.WebApi2,
+    Type: custom,
     Tags: {
       env: integration_tests,
       language: dotnet,
+      otel.library.name: CustomTracingExceptionHandler,
+      otel.status_code: STATUS_CODE_UNSET,
+      otel.trace_id: Guid_2,
       runtime-id: Guid_1,
```

Comment:
Kinda suspicious that runtime-id is Guid_2 in all other snapshots but in this one it's Guid_1 🤔 

Welp, if it passes it passes!